### PR TITLE
Adds config file entry for pad squircling factor

### PR DESCRIPTION
**Pull request by RipleyTom**
Defaut 8000, go lower to go closer to (0,0) corner.

### CODE REVIEW

**Comment by Nekotekina** [comment]
This is not core configuration.

**Comment by fifazalata** [comment]
This PR confirmed fixing diagonal input in P5
it fixes it when setting pad squircling factor to 4000 instead of the default 8000

**Comment by Nekotekina** [comment]
And this doesn't justify moving kbpad settings as well

**Comment by RipleyTom** [comment]
Fixored

File: rpcs3/xinput_pad_handler.h [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
 #define X_INPUT_PAD_HANDLER
 
+#include "Utilities/Config.h"
 #include "Emu/Io/PadHandler.h"
 #include <Windows.h>
 #include <Xinput.h>
 
+struct xinput_config final : cfg::node
+{
+	const std::string cfg_name = fs::get_config_dir() + "/xinput_config.yml";
```
Comment (Nekotekina) [inline rpcs3/xinput_pad_handler.h]: config_xinput.yml

**Pull request merged**